platform/broadcom/docker-syncd-brcm-dnx/Dockerfile.j2

```diff
@@ -23,6 +23,18 @@ RUN apt-get install -yf kmod
 ## BRCM uses ethtool to set host interface speed
 RUN apt-get install -y ethtool
 
+## Install gcc, libc6-dev and python3-dev for compiling python-lzf
+RUN apt-get -y install build-essential libc6-dev python3-dev 
+
+## Install python-lzf
+RUN pip3 install 'python-lzf==0.2.4'
+
+## Install rdbtools
+RUN pip3 install 'rdbtools==0.1.15'
+
+## Uninstall gcc, libc6-dev and python3-dev for compiling python-lzf
+RUN apt-get -y remove build-essential libc6-dev python3-dev
```

Comment:
all these should be in base docker, since rdbtool is generic

Reply:
Fix it.